cli/src/main.rs

```diff
@@ -77,24 +75,81 @@ fn run() -> Result<(), CliError> {
                 (@arg force: --force "Overwrite files if they exist")
             )
         )
-        (@subcommand cert =>
-            (about: "Generate certificates that can be used for development")
-            (@subcommand generate =>
-                (about: "Generate certificates and keys for the ca, server and client")
-                (@arg common_name: --("common-name") +takes_value
-                  "The common name that should be used in the generated cert, default localhost")
-                (@arg cert_dir: -d --("cert-dir") +takes_value
-                  "Name of the directory in which to create the certificates")
-                (@arg force: --force  conflicts_with[skip] "Overwrite files if they exist")
-                (@arg skip: --skip conflicts_with[force] "Check if files exists, generate if missing")
-            )
-        )
+    );
+
+    app = app.subcommand(
+        SubCommand::with_name("cert")
+            .about("Generate certificates that can be used for development")
+            .setting(AppSettings::SubcommandRequiredElseHelp)
+            .subcommand(
+                SubCommand::with_name("generate")
+                    .long_about(
+                        "Generates test certificates and keys for running splinterd with \
```

Comment:
Is there a reason for the `about` and `long_about` content to be different?  I recommend consistency whenever possible. 

The `long_about` info seems more helpful than the short `about` blurb, because it clarifies that the command generates keys for splinterd and mentions TLS/insecure mode. 

Also, line 82 should start with "Generates".

Reply:
I have been enlightened in an off-line chat, so I am retracting my comment.